src/main/java/duke/command/AddCommand.java

```diff
@@ -0,0 +1,61 @@
+package duke.command;
+
+import duke.DukeException;
+import duke.TaskList;
+import duke.task.DeadlineTask;
+import duke.task.EventTask;
+import duke.task.Task;
+import duke.task.ToDoTask;
+
+import java.time.LocalDate;
```

Comment:
Standard `java.` imports should be placed above `duke.` imports according to `checkstyle` rules